charts/karpenter/templates/_helpers.tpl

```diff
@@ -171,21 +171,35 @@ Flatten Settings Map using "." syntax
 {{/*
 Flatten the stdout logging outputs from args provided
 */}}
-{{- define "karpenter.controller.outputPathsList" -}}
+{{- define "karpenter.outputPathsList" -}}
 {{ $paths := list -}}
 {{- range .Values.controller.outputPaths -}}
-    {{- $paths = printf "%s" . | quote  | append $paths -}}
+    {{- if not (has (printf "%s" . | quote) $paths) -}}
+        {{- $paths = printf "%s" . | quote  | append $paths -}}
+    {{- end -}}
```

Comment:
QQ: hoping you can help educate me here. Don't we want to include these paths if they're defined? It looks like the code is saying "if not has string defined" then add it in, which I don't think makes sense. What am I missing here?

Reply:
This is saying: "If this string doesn't already exist in the set of paths, then add it." It's uniqueness criteria.

Reply:
Ahhhh gotcha. Thanks!